lib/plus_codes/open_location_code.ex

```diff
@@ -0,0 +1,489 @@
+defmodule PlusCodes.OpenLocationCode do
+  require Integer
+
+  @moduledoc """
+  Implements the Google Open Location Code(Plus+Codes) algorithm.
+  """
+
+  @separator '+'
+  @separator_position 8
+  @pair_code_length 10
+  @padding '0'
+  @code_alphabet '23456789CFGHJMPQRVWX'
```

Comment:
you probably want these values in double quotes, not single quotes

Reply:
Nope, I want a character array as I extract and lookup characters.